### PR TITLE
add script calling kaggle API and Pandas

### PR DESCRIPTION
**Pull request by marcelinaszcz95**
The script **bikes_script.ipynb** allows to fetch the dataset from the _Kaggle_, using Kaggle API.

It creates the _download_ folder, where the input CSV file is sorted, followed by data cleaning, and transformation.

In the end when the data is processed, it creates the _output_ folder, if it does not exist, and save the _output_ file there as a _london_bikes_output.xlsx_. 

The file can be then used for the visualization and further analysis.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -160,3 +160,8 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+output
+downloads
+london_bikes_final.xlsx
+*kaggle.json
```
Comment (dmlemeshko) [inline .gitignore]: please add empty line in the end

File: src/kaggle_.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"username":"your username","key":"your key"}
```
Comment (dmlemeshko) [inline src/kaggle_.json]: same here, empty line

Comment (marcelinaszcz95) [reply src/kaggle_.json]: ok

File: .gitignore (hunk 1 above)
Comment (marcelinaszcz95) [reply .gitignore]: ok

**Comment by dmlemeshko** [comment]
LGTM

**Pull request merged**